src/hooks/use-contact-group-actions.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { useCallback, useMemo } from 'react';
+
+import { type Action } from '@zextras/carbonio-design-system';
+import { useIntegratedFunction } from '@zextras/carbonio-shell-ui';
+import { useTranslation } from 'react-i18next';
+
+import { ContactGroup } from '../model/contact-group';
+
+export const useContactGroupActions = (contactGroup: ContactGroup): Action[] => {
+	const { id, title, members } = contactGroup;
+	const [t] = useTranslation();
+
+	const [openMailComposer, isMailAvailable] = useIntegratedFunction('composePrefillMessage');
```

Comment:
id and title are not used, remove them. Also, the destructuring could be done directly inside the arguments of the function